runtime.md

```diff
@@ -23,37 +19,111 @@ This allows the hooks to perform cleanup and teardown logic after the runtime de
 * **`bundlePath`**: (string) is the absolute path to the container's bundle directory.
 This is provided so that consumers can find the container's configuration and root filesystem on the host.
 
-*Example*
-
+When serialized in JSON, the format MUST adhere to the following pattern:
```

Comment:
What was the intent of this sentence? Does "the following pattern" include the indentation pattern too?
- https://github.com/opencontainers/runtime-spec/issues/1177

Reply:
commented in the issue